EMA_Feature_Comparison.json

```diff
@@ -1415,6 +1439,18 @@
                             "MetricWire - Notes": "",
                             "Movisens - Score": "Supported",
                             "Movisens - Notes": "On request. Via \"Geofence Trigger\" or \"Multi Geofence Trigger\". This event fires if a given geofence or any of the given geofences was entered/left or the participant is staying in it over some time. Other then the geofence itself, you can specify whether the geofence should trigger the initial location of the participant dependent on the geofence."
+                        },
+                        {
+                            "Row ID": "3.1.14",
+                            "Feature Name": "Sensor-based Activity Triggering - GPS (Location Change)",
+                            "Avicenna (Ethica) - Score": "Not Supported",
+                            "Avicenna (Ethica) - Notes": "",
+                            "m-Path - Score": "Not Supported",
+                            "m-Path - Notes": "",
+                            "MetricWire - Score": "Not Supported",
+                            "MetricWire - Notes": "",
+                            "Movisens - Score": "Supported",
+                            "Movisens - Notes": ""
```

Comment:
On request only. You should specify the Distance the device must travel to trigger a location change.